tests/spread/plugins/v2/snaps/catkin-tools-noetic-in-source-hello/package.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>snapcraft_hello</name>
+  <version>0.0.1</version>
+  <description>snapcraft test for roscpp</description>
+  <maintainer email="me@example.com">me</maintainer>
+  <license>Apache License 2.0</license>
```

Comment:
I know this was probably copy/paste, but the license field here doesn't match the source file headers.  Maybe should be gpl3?

Reply:
Indeed this was copy/pasted from the [catkin-noetic-hello](https://github.com/snapcore/snapcraft/blob/1df75d9d6a032523a2fa1ca06e634e50e4574dcf/tests/spread/plugins/v2/snaps/catkin-noetic-hello/src/snapcraft_hello/package.xml) test (and duplicated in [catkin-noetic-in-source-hello](https://github.com/snapcore/snapcraft/blob/1df75d9d6a032523a2fa1ca06e634e50e4574dcf/tests/spread/plugins/v2/snaps/catkin-noetic-in-source-hello/package.xml) test too). Should I fix them all?

Reply:
A follow up PR fixing them all is fine